graphql/execute.lua

```diff
@@ -260,6 +260,7 @@ local function getFieldEntry(objectType, object, fields, context)
 
   local arguments = util.map(fieldType.arguments or {}, function(argument, name)
     local supplied = argumentMap[name] and argumentMap[name].value
+    if argument.kind then argument = argument.kind end
```

Comment:
Wow, looks like a random tweak. Can you clarify it somehow?

Reply:
I've discussed the point with @no1seman . The motivation was to support 
```
arguments = {
    arg = {
        kind = types.string,
    },
},
```
input descriptions, but it  seems it may be excessive. @no1seman said he will review this additionally and provide some comment on why we should do it or it will be verified that it is unnecessary and I'll remove this

Reply:
Is it possible to investigate the problem first and then add code?

Reply:
Yes. If it doesn't affect "description for inputObjects" I suggest to move it into separate patch and discuss separately

Reply:
The promblem is that without this lines if argument is defined by:  arg = { kind = types.string, }, then validation of request on execute will fail.

Reply:
Well, if it's a bugfix it could be moved into separate patch with relevant tests and commit message

Reply:
Since this PR tests run correct even without the fix, I separated it to https://github.com/tarantool/graphql/pull/31 with corresponding tests and descriptions.